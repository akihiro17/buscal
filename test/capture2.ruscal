if capture2("d1", command("date", "+%Y-%m-%d", "-d 2025-05-18")) && capture2("d2", command("date", "+%Y-%m-%d", "-d 2024-05-18")) {
  echo("success");
  echo(d1);
  echo(d2);
} else {
  echo("fail");
}
