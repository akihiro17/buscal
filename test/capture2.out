success
2025-05-18
2024-05-18
